nixos/modules/programs/nist-feed.nix

```diff
@@ -0,0 +1,47 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.nist-feed;
+in
+{
+  options = {
+    programs.nist-feed = {
+      enable = mkEnableOption (mdDoc "NIST Feed notifies you about the newest published CVEs.");
+      package = mkPackageOption pkgs "nist-feed" { };
+      arguments = mkOption {
+        type = types.str;
+        default = "-l -s CRITICAL";
+        description = mdDoc ''lib.
+          Arguments to provide to NIST-Feed.
+        '';
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    environment.systemPackages = [ cfg.package ];
+    systemd.user.services.nist-feed = {
+        wantedBy = [ "default.target" ];
+        description = "NIST-Feed service";
+        path = [ pkgs.curl pkgs.busybox ];
+        serviceConfig = {
+          Type = "oneshot";
+          ExecStart = "${cfg.package}/bin/nist-feed ${cfg.arguments}";
+        };
+    };
+
+    systemd.user.timers.nist-feed = {
+        wantedBy = [ "default.target" ];
+        description = "NIST-Feed timer";
+        timerConfig = {
+          Unit = "nist-feed.service";
+          OnCalendar = "*:0/30";
```

Comment:
I think it would be nice if this was configurable

Reply:
I also thought the same, but then aligned with the [source](https://github.com/D3vil0p3r/NIST-Feed/blob/main/nist-feed#L348C32-L348C132)

Reply:
[Done, it is now configurable](https://github.com/NixOS/nixpkgs/pull/284812/commits/7af570531d8ccb77db2ea6fef0a92c0e9c85d7ad)